registration-processor/registration-processor-rest-client/src/main/java/io/mosip/registration/processor/rest/client/utils/RestApiClient.java

```diff
@@ -56,6 +59,12 @@ public class RestApiClient {
 	/** The logger. */
 	private final Logger logger = RegProcessorLogger.getLogger(RestApiClient.class);
 
+	@Value("${registration.processor.httpclient.connections.max.per.host:20}")
+	private int maxConnectionPerRoute;
+
+	@Value("${registration.processor.httpclient.connections.max:100}")
+	private int totalMaxConnection;
+
```

Comment:
Why do we need these variable here ? Seems like not used in the file.

Reply:
Removed as it is not used in release branch also